src/pages/users/modification/user-modification-form.tsx

```
@@ -8,6 +8,7 @@
 import { useCallback, useMemo, type FunctionComponent } from 'react';
 import { Grid2 as Grid, IconButton, Tooltip } from '@mui/material';
 import * as yup from 'yup';
+import { AutocompleteInput, TextInput, NAME_EMPTY } from '@gridsuite/commons-ui';
```

ESLint

[error] '/home/jailuser/git/node_modules/@gridsuite/commons-ui/dist/index.js' imported multiple times.

(import-x/no-duplicates)